### PR TITLE
written LABEL in key value format

### PR DESCRIPTION
**Pull request by susheelpal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1653"></a>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM node:15 as development
```
Comment (Panquesito7) [inline Dockerfile:1]: Please revert the changes in this file. Let us know if you need any help. 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Comment by susheelpal** [comment]
ok sure, will revert my changes
